hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeAdminDefaultMonitor.java

```diff
@@ -113,6 +113,17 @@ protected void processConf() {
       numBlocksPerCheck =
           DFSConfigKeys.DFS_NAMENODE_DECOMMISSION_BLOCKS_PER_INTERVAL_DEFAULT;
     }
+
+    final String deprecatedKey =
```

Comment:
This should marked too in DFSConfigKeys.

Reply:
Hi @goiri thanks for your review.
currently already removed  "dfs.namenode.decommission.nodes.per.interval"  in DFSConfigKeys ,
so no need to mark it, right?